config/config/common-config.yaml

```diff
@@ -195,6 +195,11 @@ prometheusOperator:
       cpu: 100m
       memory: 200Mi
 
+  ## Set the prometheus/alertmanager config reloader sidecar CPU request and limit
+  configReloaderCpu: 50m
+  ## Set the prometheus/alertmanager config reloader sidecar memory request and limit
+  configReloaderMemory: 50Mi
+
```

Comment:
These keys are something specific for the alertmanager/prometheus crds right?

Reply:
It's specific to the prometheus operator.

Reply:
https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/upstream/kube-prometheus-stack/templates/prometheus-operator/deployment.yaml#L73-76

Reply:
I see, just found it curious that they set limit=request through a single key

Reply:
Yep I agree, but it did not feel important enough to try to change it upstream.